lab2.js

```diff
@@ -39,32 +39,65 @@ function assert(expression, failureMessage) {
         1          |       2/hour
         2          |       3/hour
         3          |       4/hour
+        4   | 5/hour
 
  TODO: First, make a constructor function, called Blob, that makes blobs.
+*/
+function Blob(personsConsumed, rateOfConsumption){
+// this.blob = "Dowington";
+this.personsConsumed = personsConsumed;
+this.rateOfConsumption = rateOfConsumption;
+this.timeTotal=0;
+}
+// TODO: Next, create an instance of Blob named blob.
+var blob = new Blob();
+ //TODO: Then, use a loop to calculate how long it took the blob to finish
+ //with Dowington.
+var dowington = 1000;
+//take blob expansion rate n + 1
+function blobRate(hours){
+  var humansEaten = hours + 1;
+  return humansEaten;
+}
 
- TODO: Next, create an instance of Blob named blob.
 
- TODO: Then, use a loop to calculate how long it took the blob to finish
- with Dowington.
-*/
+
+var blob = new Blob(0,1);
+
+
+for (var rateOfConsumption=1; rateOfConsumption <= 1001; rateOfConsumption++ ){
+ this.timeTotal += (60 / rateOfConsumption);
```

Comment:
not really sure what's going on here? the rate of consumption is per hour (what you found in the function above), not per minute.